test/files/neg/t5265b.check

```diff
@@ -1,4 +1,9 @@
-t5265b.scala:7: error: Implicit definition must have explicit type (inferred T[String])
+t5265b.scala:7: warning: Implicit definition must have explicit type (inferred T[String])
   implicit val tsMissing = new T[String] {}   // warn val in trait
                ^
+t5265b.scala:20: warning: Implicit definition must have explicit type (inferred T[String])
+  implicit val tsChild = new T[String] {}     // nowarn because inferred from overridden
```

Comment:
dotty agrees with the previous behavior: nowarn when specified by overridden member